src/index.md

```diff
@@ -7,3 +7,8 @@ about the core specification, as well as many modality-specific extensions.
 To get started, [check out the introduction](01-introduction.md). If you'd like
 more information on how to adapt your own datasets to match the BIDS
 specification, we recommend exploring the [bids-specification starter kit](https://github.com/bids-standard/bids-starter-kit).
+
+If you are interested in getting occasional updates on BIDS 
+please consider signing up for our [mailing list](https://docs.google.com/forms/d/1ZLi5qRTuX11KGK7qIidSdZvznFoXAqr2wh6003okv-0/edit)
+
+You can also find us on Twitter! ![](http://img.shields.io/twitter/follow/bidsstandard.svg?style=social)
```

Comment:
A badge here doesn't seem to add anything.

```suggestion
You can also find us on [Twitter](https://twitter.com/BIDSstandard]!
```

Reply:
I think a badge looks better than having a hyperlink to twitter

Reply:
applying your previous suggestion to get the badge directly linked to twitter

```suggestion
You can also find us on Twitter! [![@BIDSstandard](http://img.shields.io/twitter/follow/bidsstandard.svg?style=social)](https://twitter.com/BIDSstandard)
```

Reply:
```suggestion

```